semeio/jobs/auto_correlate_observations/job.py

```diff
@@ -0,0 +1,35 @@
+from . import AutoCorrelateConfig
+from semeio.jobs.auto_correlate_observations import ValidationError
+from semeio.jobs.spearman_correlation_job.job import spearman_job
+
+
+def run(auto_correlation_config, measured_data, **records):
```

Comment:
What's records?

Reply:
Nothing besides allowing the general (and non-existent) framework to pass more records then the two we actually need. But I think it is a good idea to remove it until it actually exists 👍